dockstore-client/src/main/java/io/dockstore/client/cli/CheckerClient.java

```diff
@@ -0,0 +1,446 @@
+                }
+            }
+
+            // Download files
+            if (entry != null && checkerWorkflow != null) {
+                try {
+                    File downloadFolder = new File(currentDirectory);
+                    String entryToDownload = checkerWorkflow.getFullWorkflowPath() + ":" + version;
+                    downloadDescriptorFiles(entryToDownload, checkerWorkflow.getDescriptorType(), downloadFolder);
+                    out("Files have been successfully downloaded to the current directory.");
+                } catch (IOException ex) {
+                    exceptionMessage(ex, "Problems downloading files to " + currentDirectory, Client.IO_ERROR);
+                } catch (ApiException ex) {
+                    exceptionMessage(ex, "Problems downloading files to " + currentDirectory, Client.API_ERROR);
+                }
+            }
+        }
+    }
+
+
+    private void downloadCheckerHelp() {
+        printHelpHeader();
+        out("Usage: dockstore " + getEntryType().toLowerCase() + " download --help");
+        out("       dockstore " + getEntryType().toLowerCase() + " download [parameters]");
+        out("");
+        out("Description:");
+        out("  Downloads all checker workflow files for the given entry and stores them in the current directory.");
+        out("");
+        out("Required Parameters:");
+        out("  --entry <entry>                   Complete entry path in the Dockstore (ex. quay.io/collaboratory/seqware-bwa-workflow)");
+        out("  --version <version>               Entry version");
+        out("");
+        printHelpFooter();
+    }
+
+    private void launchChecker(List<String> args) {
+        if (containsHelpRequest(args) || args.isEmpty()) {
+            launchHelp();
+        } else {
+            // Retrieve arguments
+            String entryPath = reqVal(args, "--entry");
+
+            // Get entry from path
+            Entry entry = getDockstoreEntry(entryPath);
+
+            // Get checker workflow
+            Workflow checkerWorkflow = null;
+            if (entry != null) {
+                if (entry.getCheckerId() == null) {
+                    errorMessage("The entry has no checker workflow.",
+                        Client.CLIENT_ERROR);
+                } else {
+                    checkerWorkflow = workflowsApi.getWorkflow(entry.getCheckerId());
+                }
+            }
+
+            // Call parent launcher
+            if (entry != null && checkerWorkflow != null) {
+                // Readd entry path to call, but with checker workflow
+                args.add("--entry");
+                args.add(checkerWorkflow.getFullWorkflowPath());
+                launch(args);
+            }
+        }
+    }
+
+    private void testParameterChecker(List<String> args) {
+        if (containsHelpRequest(args) || args.isEmpty()) {
+            testParameterHelp();
+        } else {
+            // Retrieve arguments
```

Comment:
'Retrieve arguments' up to 'Get checker workflow` section eerily similar to updateVersionChecker